src/components/Map/USACountyMap.js

```diff
@@ -39,6 +56,25 @@ const USACountyMap = ({ stateClickHandler, setTooltipContent, condensed }) => {
               geographies.map(geo => {
                 const { name } = geo.properties;
                 const stateCode = getStateCode(name);
+
+                if (stateCode === 'MP') {
```

Comment:
I was originally thinking that - when I asked pablo he recommended this route since the SVG work is a bit more custom (including the CNMI text).  I don't think there is an immediately easy way to do that.